jax/_src/custom_derivatives.py

```diff
@@ -386,12 +386,15 @@ def custom_jvp_jaxpr_custom_partial_eval_rule(
   # constituent parts, so we always perform full remat. An alternative would be
```

Comment:
Return type `-> Tuple[core.JaxprEqn, core.JaxprEqn, List[bool], List[bool], List[core.Var]]` needs to be `-> Tuple[Optional[core.JaxprEqn], core.JaxprEqn, List[bool], List[bool], List[core.Var]]` with this change to avoid the type errors we're seeing here.

Reply:
You're the best, Anselm! Thanks.